src/scss/components/text-input/_index.scss

```diff
@@ -137,9 +137,10 @@ hx-text-control {
   }
 
   > p.hxErrorText {
```

Comment:
Please review file for additional design token usage.  See spacing tokens.  

https://github.com/HelixDesignSystem/helix-ui/blob/b6b0f42957c6eae9025693aab2250f6fea09d485/src/scss/components/text-input/_index.scss#L65-L92

Reply:
Changes have been done, Please review